Aplib.Core/Intent/Tactics/FirstOfTactic.cs

```diff
@@ -26,11 +26,11 @@ public FirstOfTactic(List<Tactic> subTactics, Func<bool> guard) : base(subTactic
         }
 
         /// <inheritdoc/>
-        public override List<PrimitiveTactic> GetFirstEnabledActions()
+        public override List<PrimitiveTactic> GetFirstEnabledTactics()
```

Comment:
Dit zou dan eigenlijk `GetAction` moeten worden. Maar ik dacht dat @Tboefijn dit zou fixen in een PR vandaag.